cmake_modules/commonSetup.cmake

```diff
@@ -243,6 +243,11 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   set(CMAKE_MODULE_PATH "${HPCC_SOURCE_DIR}/cmake_modules/")
 
   if(UNIX AND SIGN_MODULES)
+      execute_process(COMMAND bash "-c" "gpg --version | awk 'NR==1{print $3}'"
```

Comment:
We need to know the version of gpg we're building against because the parameters  have changed slightly from 1.4 -> 2.0 -> 2.1+